protocol/testing/e2e/funding/funding_e2e_test.go

```diff
@@ -192,31 +194,169 @@ func TestFunding(t *testing.T) {
 			oracelPriceForFundingIndex: map[uint32]string{
 				0: "27000",
 			},
-			expectedFundingPremium: 1430, // 28_000 / 27_960 - 1 ~= 0.001430
-			// 1430 / 8 * 27000 * 10^(btc_atomic_resolution - quote_atomic_resolution) ~= 483
-			expectedFundingIndex: 483,
+			expectedFundingPremiums: []perptypes.MarketPremiums{
+				{
+					PerpetualId: 0,
+					// 28_000 / 27_960 - 1 ~= 0.001430
+					Premiums: constants.GenerateConstantFundingPremiums(1430, 60),
+				},
+			},
+			// 1430 / 8 * 27000 * 10^(btc_atomic_resolution - quote_atomic_resolution) ~= 482.625
+			expectedFundingIndex: 482,
```

Comment:
for my understanding, why did `expectedFundingIndex` drop from `483` to `482`?

Reply:
This PR fixed how oracle price was updated. Previously, it was done by setting validator's local index price. This resulted in the actual oracle price being smoothed and slightly different